translations/messages-en.properties

```diff
@@ -682,5 +682,7 @@ two.weeks = 2 weeks
 three.months = 3 months
 six.months  = 6 months
 view.all = View all
+n.week = {WEEKS, plural, one{1 week} other{# weeks}}
```

Comment:
is it a convention to use capitals for "WEEKS"? 
Edit: actually it makes it clearer to the developer (like, say, me) that it's a key, and not going to be displayed, so it's a good idea. (dev UX!)